lib/rubocop/cop/rails/active_record_override.rb

```diff
@@ -48,6 +47,11 @@ def on_def(node)
 
         private
 
+        def active_model?(parent_class)
+          parent_class && %w[ApplicationRecord ActiveModel::Base]
```

Comment:
I wonder if we should add `ActiveRecord::Base` to this list to handle Rails versions lower than 5.

Reply:
And I'd probably move this list to a constant.

Reply:
>I wonder if we should add ActiveRecord::Base to this list to handle Rails versions lower than 5.

Makes sense to me!

>And I'd probably move this list to a constant.

Totally!

https://github.com/rubocop-hq/rubocop/compare/4cc3eac423a8e0f5a1ac10c62f46282221acf790..8d676717302ccba341bf12afa98318b09ec72157